### ISSUE TITLE
Acesso Curso

### ISSUE THREAD

**Issue by elemke** [da2k/curso-javascript-ninja#5035]
Comprei o curso há algum tempo, porém perdi os acessos. Como faço para recuperá-los?

@fdaciuk

**Comment by fdaciuk** [comment]
Oi @elemke! Você comprou pela Udemy?

**Comment by elemke** [comment]
Não... na época comprei diretamente pelo seu blog... Meu email: eder.lemke@gmail.com

**Comment by fdaciuk** [comment]
Oi @elemke! Eu tenho os e-mails cadastrados aqui dos alunos que tinham comprado antes da Udemy =)

Logo que foi mudado pra lá, eu mandei um e-mail avisando da mudança, e a forma de acesso. Alguns alunos reportaram que o e-mail na caixa de SPAM.

Estou reenviando pra você o e-mail. Dá uma olhada, por favor, se caiu no SPAM pra você também, e me avise se recebeu ou não, ok? =)

**Comment by fdaciuk** [comment]
Show! Qualquer coisa, só avisar =)
